karma.conf.js

```diff
@@ -37,8 +37,20 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Hopefully you don't mind, can undo, but I'm running everything via CLI so cant actually run chrome.

Reply:
I don't mind - I assume the literal visual rendering time if somewhat accounted for. Unless we get an issue that the tests with headless aren't picking up, I think that's dandy